RELEASE.md

```diff
@@ -1,69 +1,37 @@
 # How to make a release
 
-`jupyterhub-idle-culler` is a package [available on
-PyPI](https://pypi.org/project/jupyterhub-idle-culler/). These are instructions
-on how to make a release on PyPI. The PyPI release is done automatically by a
-GitHub workflow when a tag is pushed.
+`jupyterhub-idle-culler` is a package available on [PyPI][] and [conda-forge][].
+These are instructions on how to make a release.
 
-For you to follow along according to these instructions, you need:
+## Pre-requisites
 
-- To have push rights to the [jupyterhub-idle-culler GitHub
-  repository](https://github.com/jupyterhub/jupyterhub-idle-culler).
+- Push rights to [jupyterhub/jupyterhub-idle-culler][]
+- Push rights to [conda-forge/jupyterhub-idle-culler-feedstock][]
```

Comment:
Is this a requirement for our other packages too? I don't believe i have that

Reply:
No, conda-forge push rights shouldn't be necessary for any package. That process should be mostly automated and handled on the conda-forge side.

Reply:
Thank you! Okay I've connected these in the past, but I'll disconnect these responsibilities.

In [bbb3818](https://github.com/jupyterhub/jupyterhub-idle-culler/pull/58/commits/bbb38187380914fce47fc7e09a64b4d56ac257e0) I fixed some mistakes I made when copying content from jupyterhub/jupyterhub, omitting links and a final step.

I still link out to conda-forge, but write:

>1. Following the release to PyPI, an automated PR should arrive within 24 hours
   to [conda-forge/jupyterhub-idle-culler-feedstock] with instructions on
   releasing to conda-forge. You are welcome to volunteer doing this, but aren't
   required as part of making this release to PyPI.